### DISCUSSION TITLE
Improve EE translation - The dashboard

### DISCUSSION THREAD

**Discussion by Caleydon** [ExpressionEngine/ExpressionEngine#3003] [Feature Requests]
File: /system/ee/ExpressionEngine/View/_shared/dashboard/dashboard.php
Please, could you move the sentences and phrases from following part of code into translation files?

```
	<div class="dashboard__item  widget widget--support">
		<div class="widget__title-bar">
			<h2 class="widget__title">ExpressionEngine Support</h2>
		</div>

		<p>Get <b>direct</b>, <b>fast</b>, <b>unlimited</b> support from the same team that builds your favorite CMS.</p>

		<p><a href="https://expressionengine.com/support" target="_blank" class="button button--default">Learn More</a></p>
	</div>
```

**Comment by intoeetive** [comment]
@TomJaeger are we looking into making this widget translatable at all?

**Comment by Caleydon** [comment]
This block is displaying on CP home page. If the admin/user using different language as English, he will see language mixture on CP homepage.

**Comment by TomJaeger** [comment]
Yeah, we should probably make it translatable
